sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -565,16 +566,31 @@ class depends_on {
   public:
     template<typename... NodeTN>
     depends_on(NodeTN... nodes);
+
+    template<typename... EventTN>
+    depends_on(EventTN... events);
 };
 }
 ----
 
 The API for explicitly adding nodes to a `command_graph` includes a
 `property_list` parameter. This extension defines the `depends_on` property to
-be passed here. `depends_on` defines any `node` objects for the created node to
-be dependent on, and therefore form an edge with. These nodes are in addition to
-the dependent nodes identified from the command-group requisites of the created
-node.
+be passed here. `depends_on` may be used in two ways:
+
+* Passing nodes from the same `command_graph` which will create dependencies and
+graph edges between those nodes and the node being added. 
+
+* Passing SYCL events, including <<dynamic-event, Dynamic Events>>. If an event
+represents a recorded node in the same graph, then a graph edge is created
```

Comment:
```suggestion
represents a node in the same graph, then a graph edge is created
```
"recorded" may give a wrong impression that only for the recorded graph, not the graph created with explicit APIs.

Reply:
Doesn't `command_graph::get_event` provide this ability?

Reply:
> Doesn't command_graph::get_event provide this ability?

Since `get_event()` can only be called on executable graphs and returns an event specifically representing an execution point in a single graph execution (and can be used largely as normal SYCL events), they are conceptually quite different from the events returned from recorded queue submissions. 

Those have no backend events associated with them, have many restrictions on where they can be used and are only for use in creating dependencies between graph nodes.